src/client/components/DocumentList/DocumentLink.tsx

```diff
@@ -0,0 +1,161 @@
+import * as Sentry from '@sentry/react';
+import classnames from 'classnames';
+import { useCallback, useState } from 'react';
+import { GenericDocument } from '../../../universal/types';
+import { IconAlert, IconDownload } from '../../assets/icons';
+import { Colors } from '../../config/app';
+import { trackDownload, useProfileTypeValue } from '../../hooks';
+import { useUserCity } from '../../hooks/useUserCity';
+import Linkd from '../Button/Button';
+import { Spinner } from '../Spinner/Spinner';
+import styles from './DocumentLink.module.scss';
+
+interface DocumentLinkProps {
+  document: GenericDocument;
+  label?: string;
+  trackPath?: (document: GenericDocument) => string;
+}
+
+function downloadFile(docDownload: GenericDocument) {
```

Comment:
functions can be placed under the functions/classes/code that they're used in I guess

Reply:
Ik weet niet wat je hiermee bedoelt